resources/user.go

```diff
@@ -59,8 +63,31 @@ func (obj *UserRes) Default() Res {
 
 // Validate if the params passed in are valid data.
 func (obj *UserRes) Validate() error {
+	const badChars string = " \"{}()<>&*‘|=?;[]$–#~!.”%%/\\:+,`"
```

Comment:
can you either add a comment on where you got the spec for badChars, or make it obvious in a different way?
eg: whitelist of a-z + specialchars?